### PR TITLE
 Added code tags and corrected some lines

### PR DESCRIPTION
**Pull request by RaTi17**
Added code tags to parameters and changed lines to be more consistent, i.e. 'Returns' instead of 'Return' and a few other minor adjustments.
Changed the fmod description and example, added a FuncRef reference to funcref.

### CODE REVIEW

File: doc/classes/@GDScript.xml [hunk 1]
```diff
@@ -313,7 +313,7 @@
 			<argument index="0" name="s" type="float">
 			</argument>
 			<description>
-				Raises the Euler's constant [b]e[/b] to the power of 's' and returns it. [b] has an approximate value of 2.71828.
+				Raises the Euler's constant [b]e[/b] to the power of [code]s[/code] and returns it. [b] has an approximate value of 2.71828.
```
Comment (Noshyaar) [inline doc/classes/@GDScript.xml]: Unpaired [b] tag.

Comment (willt) [reply doc/classes/@GDScript.xml]: should be: [b]e[/b] has an approximate

File: doc/classes/@GDScript.xml [hunk 2]
```diff
@@ -250,7 +250,7 @@
 			<argument index="0" name="step" type="float">
 			</argument>
 			<description>
-				Returns the number of digit places after the decimal that the first non-zero digit occurs.
+				Returns the offset (after the decimal point) of the first non-zero digit. The returned offset starts at 1.
```
Comment (willt) [inline doc/classes/@GDScript.xml]: Is offset the right word to use here? I'm not sure what I would use this method for but would probably involve the total digits not the place of the digits which to me makes the use of offset seem wrong.

File: doc/classes/@GDScript.xml (hunk 1 above)
Comment (RaTi17) [reply doc/classes/@GDScript.xml]: Yep, missed that

File: doc/classes/@GDScript.xml (hunk 2 above)
Comment (RaTi17) [reply doc/classes/@GDScript.xml]: I agree, it's more something of the scale of the smallness of the fractional part. Although that doesn't sound really nice either. I'll try to make a better sentence of it without 'offset'.

Comment (RaTi17) [reply doc/classes/@GDScript.xml]: Perhaps :
Returns the position of the first non-zero digit, after the decimal point.
?

**Pull request merged**